pgcli/main.py

```diff
@@ -137,7 +137,11 @@ def __init__(self, force_passwd_prompt=False, never_passwd_prompt=False,
         self.syntax_style = c['main']['syntax_style']
         self.cli_style = c['colors']
         self.wider_completion_menu = c['main'].as_bool('wider_completion_menu')
-        self.less_chatty = c['main'].as_bool('less_chatty')
+        if less_chatty is not None:
+            self.less_chatty = less_chatty
+        else:
+            self.less_chatty = c['main'].as_bool('less_chatty') or False
```

Comment:
This line might not be intuitive enough because it relies on Python behavior of boolean logic with `None`. (See ["False or None vs. None or False"](http://stackoverflow.com/questions/3914667/false-or-none-vs-none-or-false).)

The best alternative I see is a nested `if/else` but that's verbose compared with how other settings are handled here. I was disappointed `as_bool` doesn't take a default.

I went to all this trouble in order to handle when `less_chatty` is `True`, `False`, or absent from the config file.

It's only possible to turn chattiness _off_ with the command line option. On reflection, perhaps it makes sense for `--less-chatty` to take a true/false value.